analyzers/src/SonarAnalyzer.Common/Rules/StringLiteralShouldNotBeDuplicatedBase.cs

```diff
@@ -87,9 +87,11 @@ private void ReportOnViolation(SyntaxNodeAnalysisContext context)
             {
                 var duplicates = item.ToList();
                 var firstToken = duplicates[0];
+                var messageText = new string(item.Key.Where(x => !char.IsControl(x)).ToArray()); // Strip of any control characters to not mess up the error message.
+                // alternative: var messageText = firstToken.Text.StripOfLeadingAndTrialingDoubleQuotes();
```

Comment:
Let's use the `firstToken.Text.StripOfLeadingAndTrialingDoubleQuotes()` variant. It looks way easier.

Reply:
I will restore the function I removed before in the other PR to do that:
https://github.com/SonarSource/sonar-dotnet/commit/4ae27ac41ac1f176ce6c8adb145d601f7dd4bc51#diff-532dadf2618d4537260a9966cba9f8ef0f35e56115d876894024f4211356505aL113-L114